src/display/editor/editor.js

```diff
@@ -1319,9 +1392,139 @@ class AnnotationEditor {
     if (this.isResizable) {
       this.#createResizers();
       this.#resizersDiv.classList.remove("hidden");
+      bindEvents(this, this.div, ["keydown"]);
     }
   }
 
+  /**
+   * onkeydown callback.
+   * @param {KeyboardEvent} event
+   */
+  keydown(event) {
+    if (
+      !this.isResizable ||
+      event.target !== this.div ||
+      event.key !== "Enter"
+    ) {
+      return;
+    }
+    this._uiManager.setSelected(this);
+    this.#savedDimensions = {
+      savedX: this.x,
+      savedY: this.y,
+      savedWidth: this.width,
+      savedHeight: this.height,
+    };
+    const children =
+      this.#allResizerDivs || Array.from(this.#resizersDiv.children);
+    if (!this.#allResizerDivs) {
+      this.#allResizerDivs = children;
+      for (const div of children) {
+        const name = div.getAttribute("data-resizer-name");
+        div.addEventListener("keydown", this.#resizerKeydown.bind(this));
+        div.addEventListener("blur", this.#resizerBlur.bind(this));
```

Comment:
Since these two event handlers are identical for all resizers, would it make sense to define those *once* just above the loop instead?